pom.xml

```diff
@@ -26,37 +32,111 @@
   </licenses>
 
   <build>
+    <pluginManagement>
```

Comment:
many plugins did not have their versions locked so the project could behave differently on different maven versions